### DISCUSSION TITLE
Datacamp authentication error

### DISCUSSION THREAD

**Discussion by NoumanAkhtar43** [community/community#30296] [GitHub Education]
I tried to avail datacamp free access through GitHub student pack. I faced following error. 
{"message":"no Route matched with those values"}
Talked with datacamp support about this issue and they asked for this screenshot.
![Screenshot_20220821-194500~2](https://user-images.githubusercontent.com/98735109/185796723-30be40cc-afc5-4d45-950a-3e3e7bc2e9c9.png)

Kindly help me.

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) 

Also check out these related data camp Discussions:
- https://github.com/orgs/community/discussions/30071
- https://github.com/orgs/community/discussions/29527
